### DISCUSSION TITLE
Ask for help, please

### DISCUSSION THREAD

**Discussion by DeiDeiDavid** [tw93/Pake#510] [Q&A]
Mac系统打包网页时报错如下

✵ Building app...

> pake-cli@2.1.12 tauri
> tauri build
    Updating `rsproxy` index
From https://rsproxy.cn/crates.io-index
 * [new ref]                    -> origin/HEAD
    Updating git repository `https://github.com/tauri-apps/plugins-workspace`
fatal: unable to access 'https://github.com/tauri-apps/plugins-workspace/': Recv failure: Operation timed out
error: failed to get `tauri-plugin-window-state` as a dependency of package `app v0.1.0 (/usr/local/lib/node_modules/pake-cli/src-tauri)`

Caused by:
  failed to load source for dependency `tauri-plugin-window-state`

Caused by:
  Unable to update https://github.com/tauri-apps/plugins-workspace?branch=dev#dce0f02b

Caused by:
  failed to clone into: /Users/david/.cargo/git/db/plugins-workspace-fd0259fcb18f51ad

Caused by:
  process didn't exit successfully: `git fetch --force --update-head-ok 'https://github.com/tauri-apps/plugins-workspace' '+refs/heads/dev:refs/remotes/origin/dev'` (exit status: 128)
       Error failed to build app: failed to build app
file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:369
                reject(new Error(`${code}`));
                       ^

Error: 1
    at file:///usr/local/lib/node_modules/pake-cli/dist/cli.js:369:24
    at /usr/local/lib/node_modules/pake-cli/node_modules/shelljs/src/exec.js:145:9
    at ChildProcess.exithandler (node:child_process:427:5)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:322:12)

Node.js v18.16.1

**Comment by tw93** [comment]
看起来是git clone 一个包的时候出问题了，这里你平时git clone GitHub 有问题，可以去搜索看看「如何给git设置vpn代理加速」
